source/base/thread_management.cc

```diff
@@ -15,10 +15,9 @@
 
 #include <deal.II/base/thread_management.h>
 
-#include <cerrno>
```

Comment:
Hm, do we really not need this any more? I don't recall, of course, why this was ever there, but your changes to this file at least do not remove anything that would depend on this include file, so my suspicion is that we needed it for something that remains in this file.